src/components/ContactList.tsx

```diff
@@ -82,8 +82,8 @@ const ContactList = () => {
                     )
                 })}
             </ul>
-            <Link to="/create">Create New Contact</Link>
-                <label htmlFor="contacts-search">Search UrContacts:</label>
+            <div>
```

Comment:
If the search button executes a search based on content in the input, this element should be a `<form>` and it should use an `onSubmit` rather than a button with an `onClick`
```suggestion
            <form onSubmit={handleSubmit}>
```